src/GitHook/Command/FileCommand/PreCommit/PhpStan/PhpStanConfiguration.php

```diff
@@ -40,23 +40,23 @@ public function getLevel()
      */
     public function getDirectories(): array
     {
-        return array_merge([$this->getSourceDir()], $this->getAdditionalDirs());
+        return array_merge([$this->getSourceDirectory()], $this->getAdditionalDirectories());
     }
 
     /**
      * @return string
      */
-    public function getSourceDir(): string
+    public function getSourceDirectory(): string
     {
-        return 'src/';
+        return 'src';
```

Comment:
Why removing? We always have trailing slash, see my last message on how it is done properly.